Modules/_sqlite/module.c

```diff
@@ -338,6 +344,23 @@ do {                                           \
     }                                          \
 } while (0)
 
+#define ADD_STRING(dict, key, val)                   \
```

Comment:
And ADD_STRING does not look like proper name :)

Reply:
I agree :) Will fix